flow-shop/chapter.md

```diff
@@ -248,8 +249,9 @@ To make locating the correct instance easier, we assume that every line will be
             sys.exit(0)
 ```
 
-Finally, we parse the data directly -- converting each of the tasks processing times to an integer and storing them in a list. As a final step, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above (i.e., every item in data should correspond to a particular job).
+Finally, we parse the data directly, converting each of the tasks' processing times to an integer and storing them in a list. Finally, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above. (That is, every item in `data` should correspond to a particular job.)
```

Comment:
I always shiver at the <noun>s' situation. Maybe "converting the processing time of each task to an integer..." would work?